### PR TITLE
bad permissions on description/camera/camera.py in obs_lsstSim

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

File: python/lsst/pex/config/config.py [hunk 1]
```diff
@@ -601,6 +601,17 @@ def save(self, filename, root="config"):
...
+            # for an explantion of these antics see:
+            # https://stackoverflow.com/questions/10291131/how-to-use-os-umask-in-python
+            # Get the current umask, which we can only do by setting it and
+            # then reverting to the original.
+            umask = os.umask(0o077)
+            os.umask(umask)
+            # chmod the new file to match what it would have been if it hadn't
+            # started life as a temporary file (which have more restricted
+            # permissions).
+            os.chmod(filename, (~umask & 0o666))
```
Comment (jhoblitt) [inline python/lsst/pex/config/config.py]: I will rework it to use shutil

**Comment by jhoblitt** [comment]
@timj @parejkoj take2

**Comment by parejkoj** [comment]
Did you see my comments earlier?

**Comment by timj** [comment]
I'm not entirely convinced that we need it in `utils` at this time. I'd probably make the comment though that we could be using the `utils` temporary file creator method :smile:.

**Comment by jhoblitt** [comment]
@parejkoj Yes, but I don't have time or scope to do a general refactoring of the code base for temp file handling.  This is a last minute fix before the release.

**Comment by parejkoj** [comment]
No worries.

**Pull request merged**